### ISSUE TITLE
Create Github action to publish the docker

### ISSUE THREAD

**Issue by baruchiro** [alfem/telegram-downloader-bot#17]
You can use GitHub Actions to auto publish the docker image to docker registry.

I can help you with that, but I have no "media server" (respbery PI or something), so it is currently a low priority for me.

**Comment by alfem** [comment]
Thanks for the suggestion, but I am not the author of docker files (I just coded this bot as a proof of concept). Perhaps @mitto98 could help with this.

**Comment by mitto98** [comment]
If you want, since the repo is yours, you can publish it freely on docker hub, otherwise, I can do it.

**Comment by alfem** [comment]
My laptop conked out a few days ago. If you think an image of this bot will be helpful, feel free to upload it. 

I am no a docker expert but images are architecture dependent, aren't they?

**Comment by baruchiro** [comment]
All we need is Docker Login credentials. I think @alfem should be the owner of these credentials since he is the owner of this repo.
Anyone can create the docker in the registry and give the passwords to @alfem.

And I think Github Action for publishing will be good.

**Comment by mitto98** [comment]
Yes, they are platform dependent but is possible to build arm in an x86 machine.
Tomorrow after work I'll create the image with, at least, support for x86.
I don't think is a good idea to give his docker hub credentials to strangers, but whenever you want I can give you the ownership of the docker hub repository.

**Comment by mitto98** [comment]
Since I'm not the owner of the repo I'm not allowed to add an action, but I've created a pr with some edits and a script to build and pull the image.
@alfem what do you think about adding a license?

**Comment by alfem** [comment]
Not a bad idea, @mitto98. I am not interested in earning millions of $$ with this bot :-D  so a GPL3 could work.

**Comment by mitto98** [comment]
Currently, I'm working on a university assignment, so, you know, lack of time.
I've created the repository (https://hub.docker.com/repository/docker/mitto98/telegram-download-daemon), let me know when you have finished, so I can build and push it.
I'm excited to try this script 🤩

**Comment by baruchiro** [comment]
Please create a comment [here](https://github.com/alfem/telegram-download-daemon/issues/3) so we will discuss in the right place
